Maria-Edgeworth-Letters-Project/me-tei
# Editorial note

## What does this Pull Request do?

This PR addresses the requested changes outlined in issue #34 

## How should this be tested?

Ensure that the change made to the text within editorialDecl matches expectations and that this change has been made across all XML files.

Additional changes had to be made to make all existing XML files validate. Please take a special look at Edgeworth_Bodleian_696_1819_14_17.xml. This letter had two figureDesc tags that were moved by me and might potentially not be situated as needed. I've attached the scans below. We can potentially discuss this more in our RA meeting instead and make changes after this PR is closed.
![Edgeworth_Bodleian_696_1819-04-17_page06](https://github.com/Maria-Edgeworth-Letters-Project/me-tei/assets/12545660/4d1c80ed-0973-45e2-bf3e-994eeca6bc33)
![Edgeworth_Bodleian_696_1819-04-17_page07](https://github.com/Maria-Edgeworth-Letters-Project/me-tei/assets/12545660/33f570c0-4a6b-4271-8908-e127ae1cae98)


## Additional notes

Try to avoid copying and pasting the ref values for persName and placeName. There were several instances in which a ref value went to the Personography, but a place was referenced. There are likely more that I didn't catch since this mistake doesn't cause a validation error. Please also always remember to add a file extension (.xml).
